CodeSniffer/Standards/PSR2/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -68,14 +68,53 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         }
 
         // Only one USE declaration allowed per statement.
-        $next = $phpcsFile->findNext(array(T_COMMA, T_SEMICOLON), ($stackPtr + 1));
-        if ($tokens[$next]['code'] === T_COMMA) {
+        $next = $phpcsFile->findNext(array(T_COMMA, T_SEMICOLON, T_OPEN_USE_GROUP), ($stackPtr + 1));
```

Comment:
Yes. It's a PHPCS-specific thing. PHP doesn't tokenise that code any differently, so PHPCS does it for all PHP versions.